app/views/home/_funding_progress_bar.html.erb

```diff
@@ -0,0 +1,52 @@
+<section class="progress-bar">
+  <section class="progress-bar-entry">
+    <p class="amount">$50k+</p>
+    <p class="title">Rapid Improvements</p>
+    <ul>
+      <li>Hire a second full-time dev to focus on Ruby, Rails, &amp; other projects</li>
+    </ul>
+  </section>
+
+  <section class="progress-bar-entry">
+    <p class="amount">$35-50k</p>
+    <p class="title">Steady Improvements</p>
+    <ul>
+      <li>Hire a full-time dev to improve gem tools</li>
+      <li>Finish the new index project in just a few weeks (3-6 weeks)</li>
+      <li>Migrate all of RubyGems.org to use the Fastly CDN (4-8 weeks)</li>
+      <li>Build a local gem-caching server for datacenters and offices (4 months)</li>
+      <li>Create a public Ruby ecosystem analytics dashboard (1 month)</li>
```

Comment:
Let's replace these bullet points with something like:

- Hire a full-time dev to improve gem tools
- Create a public Ruby ecosystem analytics dashboard
- Merge Bundler and RubyGems into a single, unified project
- Expand RubyBench to comprehensively cover Ruby, Rails, and Bundler

Maybe there's more, but I'm not sure.